jupyter_core/tests/test_application.py

```diff
@@ -1,12 +1,17 @@
+import os
 from tempfile import NamedTemporaryFile
 
 from traitlets import Integer
 
 from jupyter_core.application import JupyterApp
 
+pjoin = os.path.join
```

Comment:
but `pjoin` is not used anywhere here (also the `import os` would not be necessary)... maybe a leftover of a previous implementation or I am missing something?